apps/camera/locales/camera.en-US.properties

```diff
@@ -86,3 +86,11 @@ Video-Mode = Video Mode
 options = Options
 open-gallery = Open Gallery
 preview = Preview
+
+#The following strings are spoken by screen readers and not shown on the screen.
+flash-on-button.ariaLabel = Flash On
+flash-off-button.ariaLabel = Flash Off
+flash-auto-button.ariaLabel = Flash Auto
+toggle-camera-rear-button.ariaLabel = Rear camera
+toggle-camera-front-button.ariaLabel = Front camera
```

Comment:
The weird thing here is that the flash label will have the current flash mode, and the rear/front camera button will have the mode that will be set if you press the button (the opposite of the current mode). I think the rear/front needs to say "switch to rear camera".